docs/sources/datasources/prometheus.md

```diff
@@ -27,6 +27,21 @@ To access Prometheus settings, hover your mouse over the **Configuration** (gear
 | `Disable metrics lookup`  | Checking this option will disable the metrics chooser and metric/label support in the query field's autocomplete. This helps if you have performance issues with bigger Prometheus instances.           |
 | `Custom Query Parameters` | Add custom parameters to the Prometheus query URL. For example `timeout`, `partial_response`, `dedup`, or `max_source_resolution`. Multiple parameters should be concatenated together with an '&amp;'. |
 
+### Exemplars
+
```

Comment:
```suggestion

> **Note:** This feature is available in Grafana 7.4+.
```

Reply:
Or do you mean that it is not in the current Prometheus version?

Reply:
It is not in Prometheus yes.

Reply:
I've added a note with the Prometheus version should we add one with Grafana as well?

Reply:
Yes